Dynamic Priority (#135)
cap-js/notifications: CHANGELOG.md

- Support for email templates via `@notification.template.email.subject` and `@notification.template.email.html` in CDS annotations, and `EmailSubject` / `EmailHtml` in JSON templates.
- i18n support for CDS annotation string values using `{i18n>key}` syntax.
- Notification types are automatically registered and kept in sync with the notification service on application startup when running in hybrid or production mode.

> - Support for dynamic `@notification.priority` expressions on CDS events. The annotation value can be any CDS expression, including ternary operators and database functions like `days_between`. References to event payload fields are substituted at runtime and the expression is evaluated by the database. Example: `@notification.priority: (quantity > 5 ? #High : #Low)`.

Reviewer: ```suggestion
- Support for dynamic `@notification.priority` expressions on CDS events. The annotation value can be any CDS expression, including ternary operators and database functions like `days_between` (e.g.  `@notification.priority: (quantity > 5 ? #High : #Low)` ).
```
